src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -204,8 +205,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('storage blob list') as c:
         c.argument('include', validator=validate_included_datasets)
         c.argument('num_results', type=int,
-                   help='Specifies the maximum number of blobs to return. Must be in range [1,5000]. '
-                        'If this parameter is not provided, all blobs will be returned.')
+                   help='Specifies the maximum number of blobs to return. '
```

Comment:
I must have missed something, but the limitation of [1,5000] has already been lifted?

Reply:
This doc was taken from the sdk which states a limit of 5000 blobs when using `num_results`. I believe the intent was actually referring to each GET call, which does, in fact return up to 5000.
The sdk will actually page for the full number of blobs specified under the hood.